mistralrs-core/src/pipeline/hf.rs

```diff
@@ -1,7 +1,18 @@
+use signal_hook::consts::TERM_SIGNALS;
```

Comment:
Does this library work on Windows?

Reply:
Supposedly it has Windows support, but not as heavily tested.

Reply:
Let's test it on Windows before we merge it

Reply:
Tested, confirmed to work on new fix.

Reply:
fix = https://github.com/spiceai/mistral.rs/pull/24/commits/1020a5135a435da8fc7abbe1f4184080f6cca3c5